src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py

```diff
@@ -40,9 +47,15 @@ def acr_run(cmd,  # pylint: disable=too-many-locals
     _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name, RUN_NOT_SUPPORTED)
 
+    if cmd_value and file:
+        raise CLIError(
+            "Azure Container Registry can either run a contextless command or from a stream definition, but not both.")
```

Comment:
The error message needs to be actionable and point out the exact error if possible. In this case, the error message should tell the users they need to either pass `--cmd <cmd> /dev/null` or `-f acb.yaml <source_location>|/dev/null` to scheduler a run but not both.